RecoEgamma/EgammaPhotonProducers/src/PhotonProducer.cc

```diff
@@ -1,483 +1,505 @@
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("ecalRecHitSumEtOffsetBarrel"));
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("ecalRecHitSumEtSlopeBarrel"));
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("hcalTowerSumEtOffsetBarrel"));
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("hcalTowerSumEtSlopeBarrel"));
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("nTrackSolidConeBarrel"));
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("nTrackHollowConeBarrel"));
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("trackPtSumSolidConeBarrel"));
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("trackPtSumHollowConeBarrel"));
-  preselCutValuesBarrel_.push_back(conf_.getParameter<double>("sigmaIetaIetaCutBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("maxHoverEBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("ecalRecHitSumEtOffsetBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("ecalRecHitSumEtSlopeBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("hcalTowerSumEtOffsetBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("hcalTowerSumEtSlopeBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("nTrackSolidConeBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("nTrackHollowConeBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("trackPtSumSolidConeBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("trackPtSumHollowConeBarrel"));
+  preselCutValuesBarrel_.push_back(
+      conf_.getParameter<double>("sigmaIetaIetaCutBarrel"));
   //
   preselCutValuesEndcap_.push_back(conf_.getParameter<double>("minSCEtEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("maxHoverEEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("ecalRecHitSumEtOffsetEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("ecalRecHitSumEtSlopeEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("hcalTowerSumEtOffsetEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("hcalTowerSumEtSlopeEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("nTrackSolidConeEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("nTrackHollowConeEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("trackPtSumSolidConeEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("trackPtSumHollowConeEndcap"));
-  preselCutValuesEndcap_.push_back(conf_.getParameter<double>("sigmaIetaIetaCutEndcap"));
+  preselCutValuesEndcap_.push_back(
+      conf_.getParameter<double>("maxHoverEEndcap"));
```

Comment:
please recover the old setup , in part to simplify comparison of the functional changes